lib/ruby_lsp/tapioca/server_addon.rb

```diff
@@ -22,17 +23,20 @@ def execute(request, params)
             halt_upon_load_error: false,
           ).load_dsl_extensions_and_compilers
         when "dsl"
+          fork { dsl(params[:constants]) }
+        when "route_dsl"
           fork do
-            dsl(params)
+            constants = ::Tapioca::Dsl::Compilers::UrlHelpers.gather_constants
+            dsl(constants.map(&:name))
```

Comment:
If we have a different request we can take advantage of it and only run the specific compiler we are interested in. It should be faster as we'll skip `gather_constants` from other ones. It'll require more work but I suggest triggering `dsl` with a specific compiler and changing the existing logic to not delete all DSL RBIs when you specify a single compiler (This can be a different PR). Wdyt?

Reply:
I can give that a shot. I'll need to take a look at how much code is necessary to trigger only one compiler manually.

I agree, it would be a considerable improvement.

Reply:
I took a stab at this. It requires a bit of a complex refactor in Tapioca because the responsibility of generating a DSL RBI is split between the `Pipeline` and `AbstractDsl`. One generates the RBI contents and the other one further decorates it and writes to disk. I think it's an improvement we can attempt later.

However, I was successful in passing the `--only` option to DSL limiting to only 2 compilers for routes (the concerns one is necessary because it triggers some action view helpers generation).

Reply:
`--only` sounds good. Thanks.